crates/prose-xmpp/src/test/element_ext.rs

```diff
@@ -0,0 +1,92 @@
+// prose-core-client/prose-xmpp
+//
+// Copyright: 2023, Marc Bauer <mb@nesium.com>
+// License: Mozilla Public License v2.0 (MPL v2.0)
+
+use std::str::FromStr as _;
```

Comment:
What does the `as _` do here?

Reply:
Nothing, just we never use `FromStr` directly so I'm used to importing traits like this. I can revert.